EndlessClient/Controllers/ChatController.cs

```diff
@@ -85,6 +90,15 @@ public void SelectChatTextBox()
             _chatTextBoxActions.FocusChatTextBox();
         }
 
+        public void ClearAndWarnIfJailAndGlobal()
+        {
+            if (!_currentMapStateProvider.IsJail) return;
```

Comment:
Combine these `if`s to a single `if` statement

Reply:
done